sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_clients_daily_by_os_v1/query.sql

```diff
@@ -0,0 +1,94 @@
+WITH searches_per_user_by_os_and_date_staging AS (
+  SELECT
+    submission_date_s3,
+    os,
+    SUM(search_count_all) AS searches,
+    COUNT(DISTINCT client_id) AS users,
+  FROM
+    `moz-fx-data-shared-prod.telemetry.clients_daily`
+  WHERE
+    submission_date_s3 = @submission_date
+    AND app_name = 'Firefox'
+    AND sample_id = 42
```

Comment:
What would the impact be if its not filtered by sample_id. May be Romain would be able to provide insights

Reply:
I think his goal was to just do a small sample each day, since a 1% sample is probably representative and takes less processing power to query